core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -259,6 +344,7 @@ package object config {
       " over estimate, then the partitions with small files will be faster than partitions with" +
       " bigger files.")
     .longConf
+    .checkValue(_ > 0, "The number of CostInBytes event queue must not be negative")
```

Comment:
Since `_ > 0`, we should avoid 0, too.
`must not be negative` => `must be positive`.